scripts/irods/test/resource_suite.py

```diff
@@ -513,68 +513,80 @@ def test_local_iput_onto_specific_resource(self):
         if os.path.exists(datafilename):
             os.unlink(datafilename)
 
-    @unittest.skipIf(True, 'Enable once #2634 is resolved')
     def test_local_iput_interrupt_directory(self):
-        # local setup
-        datadir = "newdatadir"
-        os.makedirs(datadir)
+        local_dir_path = os.path.join(self.admin.local_session_dir, "newdatadir")
+        collection_path = os.path.join(self.admin.session_collection, 'newdatadir')
         datafiles = ["file1", "file2", "file3", "file4", "file5", "file6", "file7"]
+        restartfile = os.path.join(self.admin.local_session_dir, 'collectionrestartfile')
+        iputcmd = f'iput -X {restartfile} -r {local_dir_path} {collection_path}'
+
+        os.makedirs(local_dir_path)
         for datafilename in datafiles:
             print("-------------------")
             print("creating " + datafilename + "...")
-            localpath = datadir + "/" + datafilename
+            localpath = local_dir_path + "/" + datafilename
```

Comment:
I think I'm going to leave this as-is because trying to update something like this before in one of these older tests led to a type mismatch in some obscure interface (see https://github.com/irods/irods/pull/7583#discussion_r1537664488). It works as it is so I'm thinking I'd like to leave it. Thoughts?

Reply:
That's fine. It's not broken so let's keep it moving.

Reply:
Right-o. Resolving...